common-kafka/src/main/java/com/cerner/common/kafka/consumer/ProcessingKafkaConsumer.java

```diff
@@ -307,7 +312,9 @@ public ProcessingKafkaConsumer(ProcessingConfig config) {
 
             long currentTime = System.currentTimeMillis();
 
-            if (lastPollTime != -1L) {
+            // Exclude the first poll from the latency timer, since it takes approximately max.poll.interval.ms as the
+            // consumer group is stabilizing.
+            if (lastPollTime != -1L && pollCount > 1L) {
```

Comment:
Is there a purpose for poll count beyond excluding the first poll? If not maybe a Boolean would be a suitable replacement

Reply:
Since the latency timer is collected pre-poll and is the time since the previous call, flipping a simple boolean isn't feasible

* 1st call: pollCount = 0, we have not yet polled at all so latency is N/A. Increment to 1
* 2nd call: pollCount = 1, we have only polled once, and don't care about the latency of that one since it's the "special" initial poll. Increment to 2.
* 3rd call: pollCount = 2, we now start caring about the latency.